# FeetOutputMsg_v2.msg

time stamp
FootOutputMsg_v2 leftFoot
FootOutputMsg_v2 rightFoot
